mindjunkies/courses/signals.py

```
@@ -7,10 +7,10 @@
 
 @receiver(course_updated)
 def clear_user_cache(sender, instance, user, **kwargs):
-    cache_key = f"popular_courses_{user.uuid}"
+    cache_key = f"popular_courses"
     cache.delete(cache_key)
     print(f"Cache cleared for key: {cache_key}")
 
-    cache_key = f"new_courses_{user.uuid}"
+    cache_key = f"new_courses"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)